tests/baselines/reference/declarationEmitExpressionWithNonlocalPrivateUniqueSymbol.errors.txt

```diff
@@ -0,0 +1,11 @@
+tests/cases/compiler/b.ts(2,14): error TS2527: The inferred type of 'A1' references an inaccessible 'unique symbol' type. A type annotation is necessary.
+
+
+==== tests/cases/compiler/a.ts (0 errors) ====
+    type AX = { readonly A: unique symbol };
+    export const A: AX = 0 as any;
+==== tests/cases/compiler/b.ts (1 errors) ====
+    import { A } from './a';
+    export const A1 = A;

```

Comment:
Wait, why couldn’t it just serialize as `typeof A` (or `typeof import('./a').A` if the import gets erased) here?

Reply:
I said as why in the issue, but I'll restate it here: because that could only work for specific patterns of assignments, and while we may carve out syntactic patterns for checking JS, we do not for TS. So because it doesn't generalize beyond `var x = entity.name` assignments to all assignments, it doesn't make sense to do, really.